@kiva/kv-components/utils/loanCard.js

```diff
@@ -11,6 +11,8 @@ const SUSTAINABLE_AG_KEY = 'SUSTAINABLE AG';
 const SINGLE_PARENT_KEY = 'SINGLE PARENT';
 const REFUGEE_KEY = 'REFUGEES/DISPLACED';
 
+const findCalloutData = (tags, tagName) => tags?.find((t) => t.name.toUpperCase() === tagName.toUpperCase()) ?? {};
```

Comment:
Is `name` guaranteed to be defined or do we need chaining?

Reply:
Guaranteed; this data coming from api call.